src/Nav/index.js

```diff
@@ -1,8 +1,13 @@
 import './index.scss'
-function Nav() {
+function Nav({restaurantId, setRestaurantId}) {
+    const back = () => {
+        setRestaurantId(null)
+    }
+
     return (
-        <nav className='navbar justify-content-center navbar-expand-lg justify-content-lg-start navbar-light bg-light'>
-            <h4 className='navbar-brand my-auto mx-4'><span>Food</span>Delivery</h4>
+        <nav className='navbar justify-content-center justify-content-lg-between flex-column flex-lg-row navbar-light bg-light px-4 align-items-center'>
+            <h4 className='navbar-brand'><span>Food</span>Delivery</h4>
+            {restaurantId && <button className='' onClick={back}>&lt;&lt;Change Restaurant</button>}
```

Comment:
did you want to give the button a class? or maybe just remove all together if not